code/src/Album.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+
+export const Album = (props) => {
+    return (
+        <div className="card"> 
+          <div className="picture">
```

Comment:
Your image should be still darkened when you hover over any of the icons. (you can experiment with opacities of backgrounds of every layer)